src/style.css

```diff
@@ -1 +1,55 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(/src/fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: italic;
+  src: url(/src/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url(/src/fonts/Avenir-Heavy.ttf);
+}
```

Comment:
The `@font-face` rules define the same font-family name 'Avenir' for different styles and weights. This can lead to conflicts if the browser cannot distinguish between them. Consider using different font-family names or using the `font-weight` and `font-style` descriptors to differentiate them properly.